src/registrar/admin.py

```diff
@@ -1434,6 +1434,26 @@ def changelist_view(self, request, extra_context=None):
         # Get the filtered values
         return super().changelist_view(request, extra_context=extra_context)
 
+    def save_model(self, request, obj, form, change):
+        """
+        Override the save_model method.
+
+        On creation of a new domain invitation, attempt to retrieve the invitation,
+        which will be successful if a user exists for that email; otherwise, will
+        raise a RuntimeError, and in this case can continue to create the invitation.
+        """
+        # NOTE: is a future ticket accounting for a 'not member of this org' scenario
+        # to mirror the logic in DomainAddUser view?
+        if not change:  # Domain Invitation creation
```

Comment:
Why not just checkin the if statement,  if the email exists in the User table? That way we avoid just passing on an exception if we don't have to. Also, there is still the occasional issue where someone has multiple accounts with the same email. This scenario will cause a 500 error for the analyst that clicked the invitation, it also won't save the invitation. I would just add a check in here that it is a single email. 

Ugly suggestion just for the visual of what I am saying. This should then allow you to remove the try and except and catch the corner case of there being duplicate accounts.
```suggestion
        if not change and User.objects.filter(email=obj.email).count()==1:  # Domain Invitation creation
```

Yes the user will still get a 500 when they log in, but we do want them to contact us directly if this has happened.

Reply:
Good suggestion! Implemented this suggestion and modified the comments accordingly. Much simpler looking code now!